pkg/api/flow.go

```diff
@@ -4432,6 +4442,159 @@ func (h *flowHandler) BroadcastCloudevent(w http.ResponseWriter, r *http.Request
 
 }
 
+func (h *flowHandler) BroadcastCloudeventFilter(w http.ResponseWriter, r *http.Request) {
+
+	h.logger.Debugf("Handling request: %s", this())
+
+	ctx := r.Context()
+	namespace := mux.Vars(r)["ns"]
+	filter := mux.Vars(r)["filter"]
+
+	ces, err := ToGRPCCloudEvents(r)
+	if err != nil {
+		respond(w, nil, err)
+		fmt.Println(err)
+		return
+	}
+
+	for i := range ces {
+
+		d, err := json.Marshal(ces[i])
+		if err != nil {
+			respond(w, nil, err)
+			fmt.Println(err)
+			return
+		}
+
+		inFilter := &grpc.ApplyCloudEventFilterRequest{
+			Namespace:  namespace,
+			Cloudevent: d,
+			FilterName: filter,
+		}
+
+		rsp, err := h.client.ApplyCloudEventFilter(ctx, inFilter)
+		if err != nil {
+			respond(w, nil, err)
+			fmt.Println(err)
+			return
+		}
+
+		if string(rsp.GetEvent()) == "null" {
+			respond(w, nil, nil) // drop event if not passed filter
+			fmt.Println("dropped event")
+			return
+		}
+
+		in := &grpc.BroadcastCloudeventRequest{
+			Namespace:  namespace,
+			Cloudevent: rsp.GetEvent(),
+		}
+
+		resp, err := h.client.BroadcastCloudevent(ctx, in)
+		respond(w, resp, err)
+		return
+
+	}
+
+}
+
+func (h *flowHandler) CreateBroadcastCloudeventFilter(w http.ResponseWriter, r *http.Request) {
+	//TODO
+	h.logger.Debugf("Handling request: %s", this())
+
+	ctx := r.Context()
+	namespace := mux.Vars(r)["ns"]
+	filtername := mux.Vars(r)["filter"]
```

Comment:
Minor thing: best practices Go code uses camelcase for variables. So this should be `filterName`.